src/pages/CreateAccount/CreateAccount.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Navigate } from "react-router-dom";
+import { v4 as uuidv4 } from "uuid";
+import clsx from "clsx";
+import { ToastContainer } from "react-toastify";
+
+import {
+  setError,
+  setSuccess,
+} from "../../helpers/helperToCheckIfAccountExist";
+import { checkAllValueInObject } from "../../helpers/helper";
+
+import { ToastCreateAccount } from "../../Components/Toast";
+import Tittle from "../../Components/Tittle";
+import Input from "../../Components/Input";
+import Button from "../../Components/Button";
+
+import { createAccountUser } from "../../redux/store/usersReducer";
+
+import "react-toastify/dist/ReactToastify.css";
+
+import style from "./CreateAccount.module.scss";
+
+export function checkIfCurrentAccountExist(value) {
+  if (!value) {
+    return setError();
+  }
+  setTimeout(() => {
+    return setSuccess();
+  }, 500);
+}
+
+const CreateAccount = () => {
+  const dispatch = useDispatch();
+  const [dataToLogIn, setDataToLogIn] = useState({});
+  const [activeInp, setActiveInp] = useState(true);
```

Comment:
`activeInp` and `setActiveInp` have bad naming.
If it is some boolean value, then it should be as `isInputActive` then it mean true or false.
But if `activeInp` - then i expect some input attributes e.t.c

Reply:
but in ur case it will have bad naming as well, even if u call it like `isInputActive`.
Because u check if that variable is false, then u show notification about "Fill all fields". `U shoud change naming for that variable `